auth/README.md

```diff
@@ -263,13 +263,27 @@ country code and phone number input fields. The user is still able to edit the n
 ```java
 // Use a Bundle to hold the default number, and pass it to the Builder via setParams:
 Bundle params = new Bundle();
-params.putString(AuthUI.EXTRA_DEFAULT_PHONE, "+12345678901");
+params.putString(EXTRA_DEFAULT_PHONE, "+123456789");
 IdpConfig phoneConfigWithDefaultNumber =
         new IdpConfig.Builder(AuthUI.PHONE_VERIFICATION_PROVIDER)
                 .setParams(params)
                 .build();
 ```
 
+It is also possible to set a default country code along with a national number if a specific country
+is your app's target audience. This will take precedence over the full default phone number if both
+are provided.
+
+```java
+Bundle params = new Bundle();
+params.putString(EXTRA_DEFAULT_COUNTRY_CODE, "ca");
+params.putString(EXTRA_DEFAULT_NATIONAL_NUMBER, "23456789");
```

Comment:
This naming is not consistent:
```
EXTRA_DEFAULT_PHONE
EXTRA_DEFAULT_NATIONAL_NUMBER
```

They should both be `*_NUMBER` or `*_PHONE`

Reply:
Using *_NUMBER now. 

Do you think EXTRA_DEFAULT_PHONE_NUMBER should be EXTRA_DEFAULT_FULL_PHONE_NUMBER for clarity?